packages/iris-grid/src/sidebar/conditional-formatting/ConditionalFormattingUtils.ts

```diff
@@ -717,9 +717,25 @@ export function isDateConditionValid(condition: DateCondition, value?: string) {
     case DateCondition.IS_NOT_NULL:
       return true;
 
-    default:
-      // Proper date validation will be addressed by Issue #1108
-      return value != null && value !== '';
+    default: {
+      const [dt, tz] = (value ?? '').split(' ');
```

Comment:
One suggestion - use camel case and full words for variable names. We try to follow the Airbnb style guide – https://github.com/airbnb/javascript#naming-conventions
with a few exceptions. `e` as the `catch` block argument is fine, single-character names in loops and maps are also fine, as long as the loop body is fairly short.

In this case, these are local variables and the scope is short so it's probably fine, but I thought I'd mentioned this anyway.